extensions/resteasy-reactive/rest-jackson/deployment/src/main/java/io/quarkus/resteasy/reactive/jackson/deployment/processor/JacksonOptimizationConfig.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.resteasy.reactive.jackson.deployment.processor;
+
+import java.util.function.BooleanSupplier;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+/**
+ * Jackson optimization configuration.
+ */
+@ConfigMapping(prefix = "quarkus.jackson.optimization")
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+public interface JacksonOptimizationConfig {
```

Comment:
In order to be consistent and not mix and match things, this should be a `ConfigGroup` that is references in `JacksonBuildTimeConfig`

Reply:
Actually nevermind, that only makes sense if we move this stuff into the Jackson extension